safe_yaml.gemspec

```diff
@@ -16,4 +16,8 @@ Gem::Specification.new do |gem|
   gem.executables   = ["safe_yaml"]
 
   gem.required_ruby_version = ">= 1.8.7"
+
+  if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('3.3')
```

Comment:
This condition will not work, depending on the ruby version used to _build/publish_ the gem it will either declare the dependency or not do that, for all ruby versions used to run it. I have my doubts about the possibility of a new release, just wanted to let you know.

Since the first released version of `base64` is Ruby `>= 2.3.0`, it will also implicitly raise `required_ruby_version`

Reply:
Hmmm to you think it would make sense then for me to remove the condition and bump the required_ruby_version of this gem to `>= 2.3.0`? And then if we happen to be able to get another release of the gem that should fit the bill?

Reply:
Yes. Alternatively, replace the one base64 call being done in this gem with the `unpack` equivalent and remove base64 requires:

```rb
# Equivalent to Base64.decode64
def self.base64_decode(input)
  # TODO: Condition can be droped if Ruby >= 2.4.0
  if input.respond_to?(:unpack1)
    input.unpack1('m')
  else
    input.unpack('m').first
  end
end
```